### ISSUE TITLE
Regression related to quick view and hookDisplayProductAdditionalInfo

### ISSUE THREAD

**Issue by kpodemski** [PrestaShop/PrestaShop#21359]
#### Describe the bug

Because of this PR:
https://github.com/PrestaShop/PrestaShop/pull/15297

Module developers are unable to determine if product is refreshed in quick view context or not.

#### Expected behavior

We should be able to check wether module is displayed in quick view or not, previously it was possible.

#### Steps to Reproduce

Steps to reproduce the "good" behavior from 1.7.6:

1. Make sure you have `ps_sharebuttons` module installed and enabled
2. Edit `/themes/classic/modules/ps_sharebuttons/views/templates/hook/ps_sharebuttons.tpl`
3. Put at the beginning of the block: `<pre>{$smarty.post|@print_r}</pre>`
4. Go to homepage
5. Enable quick view on some product with attributes
6. See that there's an array:

```
[action] => quickview
[id_product] => 1
[id_product_attribute] => 1
```

7. change attribute, see that we still have an access to `[action] => quickview`

Of course this is wrong that `id_product_attribute` is not updated etc. and at the end #15297 is helping with that **but**, let's check this on 1.7.7 now:

1. Make sure you have `ps_sharebuttons` module installed and enabled
2. Edit `/themes/classic/modules/ps_sharebuttons/views/templates/hook/ps_sharebuttons.tpl`
3. Put at the beginning of the block: `<pre>{$smarty.post|@print_r}</pre>`
4. Go to homepage
5. Enable quick view on some product with attributes
6. See that there's an array:

```
 [action] => quickview
 [id_product] => 1
 [id_product_attribute] => 1
```

7. **Now the important part**, change attribute, new array:

```
[ajax] => 1
[action] => refresh
[quantity_wanted] => 1
```

As you can see `action` is different than in 1.7.6, because of this I can't as a module developer check from `.tpl` if module is in quick view or not, because of this some of modules I work with doesn't work.

PR to follow.

#### Additional information

* PrestaShop version: 1.7.7
* PHP version: N/A

**Comment by florine2623** [comment]
Hi @kpodemski ,

I did manage to reproduce your issue in PS 1.7.7.
Result I have in PS 1.7.7 :
```
Array
(
    [ajax] => 1
    [action] => refresh
    [quantity_wanted] => 1
)
```
Instead of, like it is in PS 1.7.6 :
```
Array
(
    [action] => quickview
    [id_product] => 1
    [id_product_attribute] => 1
)
```

I will add this to the debug roadmap. Thanks for the PR !

**Comment by matks** [comment]
@kpodemski I'm sorry but I fail to understand how PR https://github.com/PrestaShop/PrestaShop/pull/15297 could have created this issue. The PR only wraps a hook inside a div 🤔 I know PrestaShop has sometimes very weird trick up its sleeves, but that one would be a magical one 🌈 ?

**Comment by kpodemski** [comment]
@matks ajax, as discussed on Slack 👍🏻

**Comment by Progi1984** [comment]
Fixed by #21362
